vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -138,7 +138,6 @@ public static VimeoClient getInstance() {
 
     public static void initialize(@NotNull Configuration configuration) {
         if (sSharedInstance != null) {
-            sSharedInstance.mConfiguration.deleteAccount(sSharedInstance.mVimeoAccount);
```

Comment:
We'll still be calling `setup` with the new config object each time though, which could cause other things.
And I think we do want to delete the stored account if the config object has changed, no?

Should the logic remain but instead be wrapped in a `configuration != existing config object` check?

Reply:
@msya After speaking IRL, we discussed maybe adding an `equals()` to the `Configuration` so you can only reset things if the config actually changes.  If this won't work let's get this merged to fix the issues we're seeing in dev.